docs/repo-guides/release-nnf-sw/readme.md

```diff
@@ -183,7 +183,16 @@ that everything is current on `master` for `nnf-deploy`.
 
 12. Follow steps 6-7 from the previous section to finalize the release of `nnf-deploy`.
 
-**Please review documenation for changes you may have made**
+## Release `NearNodeFlash.github.io`
+
+**Please review and update the documentation for changes you may have made.**
+
+After nnf-deploy has a release tag you may release the documentation. Use the same steps found above in "Release Each Component". Note that the default branch for this repo is "main" instead of "master".
```

Comment:
```suggestion
After nnf-deploy has a release tag, you may release the documentation. Use the same steps found above in "Release Each Component". Note that the default branch for this repo is "main" instead of "master".
```